models/src/main/java/com/vimeo/networking2/Comment.kt

```diff
@@ -37,6 +37,8 @@ data class Comment(
     /**
      * The user who posted the comment.
      */
-    val user: User?
+    val user: User?,
```

Comment:
this model is missing a default value

Reply:
Since we keep finding missing default values, an easy way to guarantee that all DTOs have default values and that new properties added also have defaults would be to create a test that just instantiated each DTO without any constructor parameters.

Reply:
I have an upcoming tasks on writing tests for models. I'll add tests for it in that task.